content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Sourcegraph 3.41 release'
+publishDate: 2022-06-21T10:00-07:00
+description: 'Sourcegraph 3.41 introduces running batch changes server-side, file mounting for the batch change container, greater pattern tracking in Code Insights, and admin settings to restrict extensions to those authored by Sourcegraph.'
+tags: [blog, release]
+slug: 'release/3.41'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+changelogItems: 
+  - description: 'A new templating variable, batch_change_link has been added for more control over where the "Created by Sourcegraph batch change ..." message appears in the published changeset description.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/35319
+    category: Batch Chanages
+  - description: Sort and limit filters have been added to capture group insights. This gives Code Insights users more control over which series are displayed.
```

Comment:
@Joelkw @iskyOS - rewrite in active voice "Added sort and limit filters to give Code Insights users more control over which series are displayed." What are group insights?

Reply:
```suggestion
  - description: Added sort and limit filters to capture group insights. This gives Code Insights users more control over which series are displayed.
```

Reply:
I believe this is referring to insights using a "capture group", a feature of regex to find instances of text patterns. @Joelkw can you confirm?

@abschu for reference: https://docs.sourcegraph.com/code_insights/explanations/automatically_generated_data_series#regular-expression-capture-group-resources

Reply:
correct! "Capture group" is the phrase, not the verb "capture" plus noun "group."